tools/add_triage_signature.py

```diff
@@ -594,11 +594,18 @@ def _prepare_output(cls, output):
         if len(output) == 0:
             return output
 
-        if len(output) > cls.TRUNCATE_OUTPUT_LENGTH:
-            half = cls.TRUNCATE_OUTPUT_LENGTH // 2
-            output = f"{output[:half]} ... **OUTPUT HAS BEEN TRUNCATED** ... {output[-half:]}"
+        pre = "{code:borderStyle=none|bgColor=transparent} "
+        post = " {code}"
+        output_list = output.replace("|", "¦").split("\\n")
 
-        return "{code:none}%s{code}" % (output.replace("|", "¦"))
+        if len(output_list) > cls.TRUNCATE_OUTPUT_LINES:
+            half = cls.TRUNCATE_OUTPUT_LINES // 2
+            output_list = output_list[:half] + ["**OUTPUT HAS BEEN TRUNCATED**"] + output_list[-half:]
+
+        output_list = list(filter(lambda x: x != "\\\"", output_list))
```

Comment:
nit: `list(filter(lambda x:` can be expressed more "elegantly" with a list comprehension `if` filter:

`[line for line in output_lines if line != "\\\""]`

Reply:
I actually think the lambda syntax is nicer, but just matter of style I guess.